app/repositories/maps.py

```diff
@@ -2,9 +2,13 @@
 
 import textwrap
 from typing import Any
+from typing import cast
 from typing import Optional
+from typing import TypedDict
 
 import app.state.services
+from app._typing import UNSET
```

Comment:
huh?

Reply:
If you look at the implementation in discord.py you can see an example for yourself:

```py
class _MissingSentinel:
    __slots__ = ()

    def __eq__(self, other) -> bool:
        return False

    def __bool__(self) -> bool:
        return False

    def __hash__(self) -> int:
        return 0

    def __repr__(self):
        return '...'


MISSING: Any = _MissingSentinel()
```


```py
def func(attr: some_type = MISSING)
```

Reply:
> UNSET and Unset being the difference between the type and the value is not a great approach

not a strong opinion but imo `Unset` is more clear/simple - and is the python standard casing for these vars - `ClassCase` and `CONST_CASE`